cdk/python/elasticloadbalancingv2-resources/tests/unit/test_elasticloadbalancingv2_resources_stack.py

```diff
@@ -0,0 +1,15 @@
+import aws_cdk as core
+import aws_cdk.assertions as assertions
+
+from elasticloadbalancingv2_resources.elasticloadbalancingv2_resources_stack import Elasticloadbalancingv2ResourcesStack
+
+# example tests. To run these tests, uncomment this file along with the example
+# resource in elasticloadbalancingv2_resources/elasticloadbalancingv2_resources_stack.py
+def test_sqs_queue_created():
```

Comment:
Delete the default test or write tests that are relevant to the sample

Reply:
will update them and push again.

Reply:
Added new commit with requested changes.